server/metrics.go

```diff
@@ -134,7 +121,7 @@ func Timed(handler http.Handler, name string, p provider.Provider) *Timer {
 func (t *Timer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if !t.isProm {
 		defer func(start time.Time) {
-			t.Observe(time.Since(start))
+			t.Observe(float64(time.Since(start).Nanoseconds()) / 1000000.0)
```

Comment:
I think the new Go-kit interface expects a float64 based on seconds not milliseconds so this should do:

```go
t.Observe(time.Since(start).Seconds())
```